CONTRIBUTING.md

```diff
@@ -123,11 +123,8 @@ $ docker-compose down
 ### Update the version and changelog
 
 1. Create a new branch for the version bump.
-1. Based on the unreleased content, determine the new version number and update
-    the [version.rb](lib/conjur-api/version.rb) file.
 1. Commit these changes - `Bump version to x.y.z` is an acceptable commit message - and open a PR
-   for review. Your PR should include updates to `lib/conjur-api/version.rb`, and
```

Comment:
I think this entire `Releasing` section needs some more attention. The project was recently added to the new automated release flow, but this section was never updated to reflect that.

Reply:
Agreed. I just removed one inaccuracy but it definitely needs a larger overhaul.